index.html

```diff
@@ -1 +1,152 @@
+                        Lexus Korea, EXR, SKT, and Supercell. Aside from the websites and their mobile variations,
+                        content I designed included images and videos,
+                        was often interactive and was used in online commercials and social media posts
+                    </p>
+                    <h3>Mountain Design Studio — Art Director</h3>
+                    <h4>Jun–Nov 2015 | Seoul, Korea
+                    </h4>
+                    <p>
+                        Directed brand identity and guidelines for 2 brands LUFT and I WEAR JUICE,
+                        including product designs,
+                        print designs,
+                        and photography
+                    </p>
+                </div>
+                <div class="inner-col" id="tool">
+                    <h2>Tools</h2>
+                    <h3>Prototyping Tools</h3>
+                    <ul>
+                        <li>Sketch</li>
+                        <li>Figma</li>
+                        <li>Invision</li>
+                        <li>Principle</li>
+                        <li>Adobe XD</li>
+                    </ul>
+                    <h3>Graphic Tools</h3>
+                    <ul>
+                        <li>Photoshop</li>
+                        <li>Illustrator</li>
+                        <li>Indesign</li>
+                        <li>Premiere Pro</li>
+                        <li>After Effects</li>
+                        <li>Cinema 4D</li>
+                    </ul>
+                    <h3>Web Development</h3>
+                    <ul>
+                        <li>HTML</li>
+                        <li>CSS</li>
+                    </ul>
+                </div>
+            </div>
+            <div class="column" id="eas">
+                <div class="inner-col" id="edu">
+                    <h2>Education</h2>
+                    <h3>Maryland Institute College of Art </h3>
+                    <h4>2019–Present | Baltimore, MD</h4>
+                    <ul>
+                        <li>MFA candidate in Graphic Design (Expected May 2021)</li>
+                        <li>Merit Scholarship Recipient</li>
+                        <li>Graduate Teaching Intern for Photo Imaging and Design for User Experience</li>
+                    </ul>
+                    <h3>Kookmin University </h3>
+                    <h4>2010–2014 | Seoul, Korea BFA in Painting & Graphic Design</h4>
+                    <p>Honors Scholarship (Spring 2011, Fall 2012)</p>
+                    <h3>Coventry University </h3>
+                    <h4>Fall 2011 | Coventry, UK</h4>
+                    <p>Exchange Program in Fine Arts </p>
+                </div>
+                <div class="inner-col" id="award">
+                    <h2>Awards</h2>
+                    <h3><a href="https://www.ux-design-awards.com/projekte/10bytes">User Experience Design Awards |
```

Comment:
I _love_ that you have all this content to link to -- but there is no visual indication to a user that there are _any_ links on your resume. Keep in mind that most users expect text links, in particular, to have either underlines or some other visual styling to make them evident.